test/ctia/http/generative/fulltext_search_test.clj

```diff
@@ -319,65 +319,68 @@
                       gen/generate)
             ignore-ks  [:created :groups :id :incident_time :modified :owner :timestamp]]
 
-        ;; to avoid accidentally picking up previous state
-        (store/delete-search incidents-store
-                             {:query "intrusion event 3\\:19187\\:7 incident"
-                              :REALLY_DELETE_ALL_THESE_ENTITIES true}
-                             login nil)
-        (is (-> incidents-store
-                (store/query-string-search
-                 {:query "intrusion event 3\\:19187\\:7 incident"}
-                 login {})
-                :data
-                empty?))
+        (assert (empty?
```

Comment:
turned it into an assert, if the state is not clean, it's useless to run this test.